browser/src/components/FoiaList.js

```diff
@@ -26,19 +37,24 @@ const FoiaList = (props) => { const [status, setStatus] = useState();
         <p className="foia-list__filters-heading">Refine results</p>
         <form className="foia-list__filters-form">
           <label htmlFor="foia-list-statuses">Status: </label>
-          <select id="foia-list-statuses" onChange={event => setStatus(event.target.value)}>
+          <select id="foia-list-statuses" onChange={event => setFilters({...filters, status: event.target.value})}>
             <option value="" key="no-status">Select a status</option>
             {filterData.statuses.map(status => (
               <option value={status} key={status}>{status}</option>
             ))}
           </select>
           <label htmlFor="foia-list-prices">Price: </label>
-          <select id="foia-list-prices" onChange={event => setPrice(parseFloat(event.target.value))}>
-            <option value="-0.1" key="no-price">Select a price</option>
+          <select id="foia-list-prices" onChange={event => setFilters({...filters, price: event.target.value})}>
+            <option value="" key="no-price">Select a price</option>
             {filterData.prices.map(price => (
               <option value={price} key={price}>{new Intl.NumberFormat(navigator.language, {style: 'currency', currency: 'USD'}).format(price)}</option>
             ))}
           </select>
+          <label htmlFor="foia-list-turnaround">Turnaround time:</label>
+          <select id="foia-list-turnaround" onChange={event => setFilters({...filters, datetime_done: event.target.value})}>
+            <option value="">Select a date…</option>
+            <option value="completed">Completed</option>
+          </select>
```

Comment:
I'm not sure I follow what this is intending to do. I recall you mentioned this PR is in an interim step, so I'm wondering if this is intended to work, or if it's a known loose end that we're fine leave undone for the moment.

Reply:
> This paves the way for adding the turn around time filter.

Ah! I just saw this PR comment. (I checked the PR out from the gh CLI so I was just looking at the code…)

I take this to mean that we don't expect this filter to work yet, so looks good to merge.

Reply:
Yes sorry about that, I should have cleaned up this PR more to extract other work in progress bits. I'll merge to keep moving, and keep in mind to clean up in the future  to reduce confusion.

Thanks for the review!